redirects.yaml

```
@@ -41,6 +41,7 @@
 /docs/patterns/link: /docs/patterns/links
 /docs/patterns/top: /docs/patterns/links#back-to-top
 /docs/patterns/list: /docs/patterns/lists
+/docs/patterns/labels: /docs/patterns/chips
```

It's `/chip`, we are a bit inconsistent in naming pages unfortunately

```suggestion
/docs/patterns/labels: /docs/patterns/chip
```